README.md

```diff
@@ -13,24 +13,25 @@
 
 ## About The Project
 
-This is the Microverse React Calculator project. The project is designed to simulate a
-real world company needing assistance with creating a calculator. The project uses Node, React, React-DOM,
-CSS, and ES6. Deployment is done via Heroku using BIG.js.
+This is the Microverse React Redux Bookstore CMS Project. The project is designed to simulate a
+real world Bookstore. The project uses Node, React, React-DOM, Redux, CSS, and ES6. 
+Deployment is done via Heroku.
 
 ## Screen Shot
 
 ![screenshot](/screen.jpg)
```

Comment:
I believe you're aware that you don't have `screen.jpg` file.
Perhaps this is just the first milestone so we don't really need it for now.